v2/types.go

```diff
@@ -85,47 +85,52 @@ type Plan struct {
 	// facing content and display instructions.  Metadata may contain
 	// platform-conventional values.  Optional.
 	Metadata map[string]interface{} `json:"metadata,omitempty"`
-	// ParameterSchemas requires a client API version >=2.13.
+	// Schemas requires a client API version >=2.13.
 	//
-	// ParameterSchemas is a set of optional JSONSchemas that describe
+	// Schemas is a set of optional RequestResponseSchema that describe
 	// the expected parameters for creation and update of instances and
 	// creation of bindings.
-	ParameterSchemas *ParameterSchemas `json:"schemas,omitempty"`
+	Schemas *Schemas `json:"schemas,omitempty"`
 }
 
-// ParameterSchemas requires a client API version >=2.13.
+// Schemas requires a client API version >=2.13.
 //
-// ParameterSchemas is a set of optional JSONSchemas that describe
+// Schemas is a set of optional RequestResponseSchema that describe
 // the expected parameters for creation and update of instances and
 // creation of bindings.
-type ParameterSchemas struct {
-	ServiceInstances *ServiceInstanceSchema `json:"service_instance,omitempty"`
-	ServiceBindings  *ServiceBindingSchema  `json:"service_binding,omitempty"`
+type Schemas struct {
+	ServiceInstance *ServiceInstanceSchema `json:"service_instance,omitempty"`
+	ServiceBinding  *ServiceBindingSchema  `json:"service_binding,omitempty"`
 }
 
 // ServiceInstanceSchema requires a client API version >=2.13.
 //
 // ServiceInstanceSchema represents a plan's schemas for creation and
 // update of an API resource.
 type ServiceInstanceSchema struct {
-	Create *InputParameters `json:"create,omitempty"`
-	Update *InputParameters `json:"update,omitempty"`
+	Create *RequestResponseSchema `json:"create,omitempty"`
```

Comment:
Should this really be `RequestResponseSchema`? The response part doesn't seem like it has meaning here.

Reply:
there is a field for each ServiceInstanceSchema field that is like: `Create.Parameters` which is old and `Create.Response` which is new. Parameters is the poorly named thing now.

This is correct.

Reply:
I believe @pmorie is referring to the fact that the response schema only makes sense for bindings, not for provisioning services. I had the same concern [here](https://github.com/openservicebrokerapi/servicebroker/pull/392#issuecomment-361955918), but @n3wscott said that he added the response schema to the provision operation [intentionally](https://github.com/openservicebrokerapi/servicebroker/pull/392#issuecomment-367765608).

I agree with @n3wscott on `parameters` now being poorly named. If we could rename that to `request`, we'd also have the option to have `RequestSchema` and `RequestResponseSchema` types, if we decide to not allow brokers to specify the response schema for provision operations.